src/ontology/mondo-edit.obo

```diff
@@ -544744,6 +544772,24 @@ xref: NCIT:C34629
 is_a: MONDO:0021201 {source="NCIT:C34629"} ! skin infection
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8306" xsd:anyURI
 
+[Term]
+id: MONDO:0100597
+name: intellectual disability, autosomal recessive
+def: "A broad category of disorders characterized by an impairment to the intelligence an individual possesses, caused by an autosomal recessive genetic disorder." [https://clinicalgenome.org/affiliation/40006/]
+is_a: MONDO:0001071 {source="https://clinicalgenome.org/affiliation/40006/"} ! intellectual disability
+intersection_of: MONDO:0001071 ! intellectual disability
+intersection_of: has_characteristic HP:0000007 ! Autosomal recessive inheritance
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8327" xsd:anyURI
+
+[Term]
+id: MONDO:0100598
+name: autosomal recessive syndromic intellectual disability
+def: "Autosomal recessive form of syndromic intellectual disability." [MONDO:patterns/autosomal_recessive]
+is_a: MONDO:0100597 ! intellectual disability, autosomal recessive
```

Comment:
A source is needed for this axiom

Reply:
I think we can probably use the ClinGen Affiliation

Reply:
I added this